lmdeploy/vl/model/llava.py

```diff
@@ -33,8 +33,7 @@ def _clip_vision_tower_load_model(self, **kwargs):
                               CLIPVisionModel)
     self.image_processor = CLIPImageProcessor.from_pretrained(
         self.vision_tower_name)
-    config = CLIPVisionConfig.from_pretrained(self.vision_tower_name,
-                                              trust_remote_code=True)
+    config = CLIPVisionConfig.from_pretrained(self.vision_tower_name)
```

Comment:
why remove `trust_remote_code=True`?

Reply:
Non AutoConfig, no need to add trust_remote_code argument